test/unit/models/instance_spec.js

```diff
@@ -154,4 +154,4 @@ describe("The Instance model", function () {
 		true
 	);
 
-});
+});
```

Comment:
It looks like you didn't make any changes to this file. You should consider using an editor with [EditorConfig](http://editorconfig.org/) support so that this sort of thing is handled automatically (the project has an `.editorconfig` file).